[2.x] Fix: Modal in mobile view

raynirola opened the pull request:
### Arranged action buttons into `flex-col` for mobile.

##### From this
<img width="368" alt="button default" src="https://user-images.githubusercontent.com/54945029/101758343-3c239080-3afe-11eb-9264-405925327df2.png">

##### To this
<img width="368" alt="Button fixed" src="https://user-images.githubusercontent.com/54945029/101758313-3463ec00-3afe-11eb-916d-3ae045d4e232.png">

### Added `break-all` to AccessToken Flash.

##### Fixes from this
<img width="368" alt="Token out of screen" src="https://user-images.githubusercontent.com/54945029/101758933-f6b39300-3afe-11eb-8ef5-9c1dbe75667a.png">

##### To this
<img width="368" alt="Fixed token flash" src="https://user-images.githubusercontent.com/54945029/101758962-016e2800-3aff-11eb-9704-d64774eac24b.png">

And many little changes, all for mobile.

taylorotwell commented:
What are the "many little changes"?

taylorotwell commented:
Re-submit when every "little change" is listed and described.

taylorotwell commented:
<img width="373" alt="image" src="https://user-images.githubusercontent.com/463230/101962623-46af5a00-3bd2-11eb-92fc-9b6633f8126e.png">

Still looks like this in Inertia?

raynirola commented:
Fixed for Interia.


### All the changes made:
- `dialog-model` added `flex layout to column for mobile`
- `confirms-password` removed `ml-2` and changed in to `sm:ml-2`
- `button` added `flex items-center justify-center`
- `secondary-button` added `flex items-center justify-center`
- `api-token-manager` removed `ml-2` and changed to `md:ml-2` 
- `delete-user-form` removed `ml-2` and changed to `sm:ml-2`
- `logout-other-browser-session` removed `ml-2` and changed in to `sm:ml-2`
- `two-factor-authentication-form `ml-2` and changed in to `sm:ml-2`

taylorotwell commented:
Your space-x on dialog modals is messing up button spacing. There is way too much space between them now. I think you are trying to do too much in this PR. Just fix one problem per PR. For example, fix the modal button stacking on mobile only in a single PR.

<img width="561" alt="Screen Shot 2020-12-13 at 2 54 28 PM" src="https://user-images.githubusercontent.com/463230/102023740-23f87f00-3d53-11eb-801c-9e75ebf0fa54.png">
